core/src/ast_builder/expr/aggregate.rs

```diff
@@ -1,25 +1,76 @@
 use {
     super::ExprNode,
     crate::{
-        ast::{Aggregate, Expr},
+        ast::{Aggregate, CountArgExpr, Expr, ToSql},
+        parse_sql::parse_expr,
         result::{Error, Result},
+        translate::translate_expr,
     },
 };
 
 #[derive(Clone)]
 pub enum AggregateNode {
+    Count(CountExprNode),
     Sum(ExprNode),
     Min(ExprNode),
     Max(ExprNode),
     Avg(ExprNode),
     Variance(ExprNode),
 }
 
+#[derive(Clone)]
+pub enum CountExprNode {
+    Text(String),
+}
+
+impl From<&str> for CountExprNode {
+    fn from(count_arg: &str) -> Self {
+        Self::Text(count_arg.to_owned())
+    }
+}
+
+impl From<ExprNode> for CountExprNode {
+    fn from(expr_node: ExprNode) -> Self {
+        match expr_node {
+            ExprNode::Identifier(str) => {
+                if str == *"*" {
+                    CountExprNode::Text(CountArgExpr::Wildcard.to_sql())
+                } else {
+                    CountExprNode::Text(CountArgExpr::Expr(Expr::Identifier(str)).to_sql())
+                }
+            }
+            _ => CountExprNode::Text(CountArgExpr::Wildcard.to_sql()),
+        }
+    }
+}
+
+impl TryFrom<CountExprNode> for CountArgExpr {
```

Comment:
AST target enum name is `CountArgExpr`, then by only adding postfix `Node`, it would be clear to understand with `CountArgExprNode`.